apps/web-giddh/src/app/new-inventory/component/aside-create-reason/aside-create-reason.component.scss

```diff
@@ -0,0 +1,65 @@
+#closedialog {
```

Comment:
check this and try to write less code for this 

please latest aside menu based on mat dailog